removed-features.asc

```diff
@@ -0,0 +1,77 @@
+:docinfo1:
+:experimental:
+
+= Removed Features
+:sectanchors:
+
+== Automatic night mode color inversion
+
+NOTE: Removed in AnkiDroid 2.17
+
+=== Feature Description
+
+AnkiDroid contained a very basic color inverter that e.g. changes white to black and black to white when night mode is enabled.
+This existed
+
+=== Reason for removal
+
+This feature was introduced in 2012, before either Android or Anki had night mode functionality.
+
+AnkiDroid implemented Android's 'follow system' night mode functionality in 2.16, which made the functionality more
+prominent.
+
+In 2.17, AnkiDroid was updated to use unified night mode card rendering code across the Anki ecosystem.
+As color inversion was an unexpected feature for our users, and did not match Anki Desktop's behavior, it was removed.
+
+https://github.com/ankidroid/Anki-Android/issues/14893
+
+=== Action to take
+
+If you are impacted by these changes, you may link:manual.html#customizingCardLayout[customize the CSS] of
+your card templates using the `night_mode` selector.
```

Comment:
I like the idea of mentioning that CSS can do an invert for people that really want it to look exactly as before. Is that possible?